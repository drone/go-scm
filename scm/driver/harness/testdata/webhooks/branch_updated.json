{
  "trigger": "branch_updated",
  "repo": {
    "id": 68,
    "path": "vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync",
    "uid": "abhinav-git-sync",
    "default_branch": "master",
    "git_url": "https://git.harness.io/vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync.git"
  },
  "principal": {
    "id": 59,
    "uid": "ec9UfvFwTf663F47Hlqxbg",
    "display_name": "abhinav.singh@harness.io",
    "email": "abhinav.singh@harness.io",
    "type": "user",
    "created": 1697617589873,
    "updated": 1697617589873
  },
  "ref": {
    "name": "refs/heads/master",
    "repo": {
      "id": 68,
      "path": "vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync",
      "uid": "abhinav-git-sync",
      "default_branch": "master",
      "git_url": "https://git.harness.io/vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync.git"
    }
  },
  "sha": "92e21bfcddc1418079cddbb518ad6fd72917798a",
  "commit": {
    "sha": "92e21bfcddc1418079cddbb518ad6fd72917798a",
    "message": "Create asdsad (#2)",
    "author": {
      "identity": {
        "name": "abhinav.singh@harness.io",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2023-12-05T11:59:39Z"
    },
    "committer": {
      "identity": {
        "name": "Harness",
        "email": "noreply@harness.io"
      },
      "when": "2023-12-05T11:59:39Z"
    }
  },
  "head_commit": {
    "sha": "92e21bfcddc1418079cddbb518ad6fd72917798a",
    "message": "Create asdsad (#2)",
    "author": {
      "identity": {
        "name": "abhinav.singh@harness.io",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2023-12-05T11:59:39Z"
    },
    "committer": {
      "identity": {
        "name": "Harness",
        "email": "noreply@harness.io"
      },
      "when": "2023-12-05T11:59:39Z"
    }
  },
  "old_sha": "a273c385628167932e10caaa58e12550c491f241",
  "forced": false
}