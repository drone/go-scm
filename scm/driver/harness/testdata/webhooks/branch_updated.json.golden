{
    "Ref": "refs/heads/master",
    "Before": "a273c385628167932e10caaa58e12550c491f241",
    "After": "92e21bfcddc1418079cddbb518ad6fd72917798a",
    "Repo": {
        "ID": "68",
        "Namespace": "",
        "Name": "abhinav-git-sync",
        "Perm": null,
        "Branch": "master",
        "Private": false,
        "Clone": "https://git.harness.io/vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync.git",
        "CloneSSH": "",
        "Link": "https://git.harness.io/vpCkHKsDSxK9_KYfjCTMKA/default/Repo_With_PR_checks/abhinav-git-sync.git",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
    "Commit": {
        "Sha": "92e21bfcddc1418079cddbb518ad6fd72917798a",
        "Message": "Create asdsad (#2)",
        "Author": {
            "Name": "abhinav.singh@harness.io",
            "Email": "abhinav.singh@harness.io",
            "Date" : "0001-01-01T00:00:00Z",
            "Login": "",
            "Avatar": ""
        },
        "Committer": {
            "Name": "Harness",
            "Email": "noreply@harness.io",
            "Date": "0001-01-01T00:00:00Z",
            "Login": "",
            "Avatar": ""
        },
        "Link": ""
    },
    "Sender": {
        "ID": "ec9UfvFwTf663F47Hlqxbg",
        "Login": "ec9UfvFwTf663F47Hlqxbg",
        "Name": "abhinav.singh@harness.io",
        "Email": "abhinav.singh@harness.io",
        "Avatar": "",
        "Created": "2023-10-18T13:56:29.873+05:30",
        "Updated": "2023-10-18T13:56:29.873+05:30"
    }
}